ttc_api_scraper.py

```diff
@@ -429,12 +437,32 @@ def archive(ctx, month, end_month):
             archive.archive_month(DBArchiver.format_month(year, mm))
     LOGGER.info('Archiving complete.')
 
+def handler(event, context):
+    """Entry point for the AWS Lambda way of launching this script"""
+
+    CONFIG = configparser.ConfigParser(interpolation=None)
+    CONFIG.read('db.cfg')
+
+    if os.environ.get('S3_BUCKET') is None:
+        LOGGER.critical("S3_BUCKET environmental variable is not set")
+        sys.exit(1)
+
+    bucket_name = os.environ.get('S3_BUCKET')
+
+    writer = WriteS3(bucket_name)
+
+    scraper = TTCSubwayScraper(LOGGER, writer, False)
+
+    loop = asyncio.get_event_loop()
+    future = asyncio.ensure_future(scraper.query_all_stations_async(loop))
+    loop.run_until_complete(future)
+
 def main():
     #https://github.com/pallets/click/issues/456#issuecomment-159543498
     cli(obj={})
 
 if __name__ == '__main__':
-
+    main()
```

Comment:
Was this intentional to call `main` 2x?

Reply:
This doesn't call main twice. It this determines whether this script was imported or called directly. It's put at the end so all the functions defined earlier are registered before we execute.

Reply:
I think @perobertson was trying to draw attention to the `try: main() except Exception:` block 👇 which would also get called after `main()`

Reply:
oh! Haha oops, sorry that got cutoff by the diff. Huh I don't even remember adding that, but yeah that shouldn't be there. Thanks! I'll take out the first one I don't even remember adding it, the generic try/except was there before and that isn't the entry point for the serverless flow